podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2017  The Hyve and respective contributors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the file LICENSE in the root of this repository.
+ */
+
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { EventManager, JhiLanguageService } from 'ng-jhipster';
+
+import {RequestService} from '../../shared/request/request.service';
+import {RequestBase} from '../../shared/request/request-base';
+import {MessageService} from '../../shared/message/message.service';
+import {Message} from '../../shared/message/message.model';
+
+@Component({
+    selector: 'request-form-submit-dialog',
+    template: `<form name="submitForm" (ngSubmit)="confirmSubmit(request)">
```

Comment:
Beforehand you don't know the size of the template when only looking a component.